assemblies/assembly_openshift-authoring-managed/main.adoc

```diff
@@ -23,6 +23,17 @@ and other business assets.
 
 * {OPENSHIFT} version 3.11 is deployed.
 * At least four gigabytes of memory are available in the OpenShift cluster/namespace.
+* For a high-availability deployment, the following resources are available on the OpenShift cluster:
+** For the {CENTRAL} replicated pod, 8 gigabytes of memory and 2 CPU cores are required per replica and two replicas are created by default.
```

Comment:
do not use per, use for each (IBM SG) Search for.

Reply:
Please make this change. We need to adhere to the SG.